### ISSUE TITLE
What do the Metrics and Losses mean?

### ISSUE THREAD

**Issue by kuboko-jp** [openai/jukebox#228]
Hi. Thank you for publishing this wonderful program.
I have a question about Metrics and loss.
I have checked the training of the 1b_lyric model in train.py through Tensorboard.
In Tensorboard, I can monitor things like loss, but I couldn't figure out what each value indicates.

For example, in prior/autoregressive.py, you will find the code shown below, and I understand that it is a cross entropy loss, but I did not understand what exactly these values indicate.
`prime_loss = F.cross_entropy(x_prime, x_t[:, :self.prime_len].reshape(-1)) / np.log(2.)`
`gen_loss = F.cross_entropy(x_gen, x_t[:, self.prime_len:].reshape(-1)) / np.log(2.)`

Could you please tell me the meaning of the values shown below as they are in [TensorBoard_training.pdf](https://github.com/openai/jukebox/files/6397617/TensorBoard_training.pdf) ?

- bpd
- gen_loss
- gn
- lg_loss_scale
- loss
